wgsl/index.bs

```diff
@@ -3838,6 +3838,8 @@ which support the WebGPU {{GPUTextureUsage/STORAGE_BINDING}} usage.
 These texel formats are used to parameterize the storage texture types defined
 in [[#texture-storage]].
 
+Note that `bgra8unorm` can only be used when the extension `bgra8unorm_storage` is enabled.
```

Comment:
Why is there this note? It seems a bit randomly placed.

Reply:
@dneto0 maybe you have a better idea about how to integrate this note. I do feel it's useful because otherwise only the extension definition describes that you need the extension in order to use bgra8unorm.

Reply:
Wait, hadn't we previously decided bgra8unorm storage would be unconditionally allowed in WGSL, and enforced only on the API side?
https://github.com/gpuweb/gpuweb/pull/3640/commits/7c474970fa928c72d831380a7f092d307e8556f2

Another discussion here reminded me of this. It seems better, for the same reason pointed out there.
https://github.com/gpuweb/gpuweb/issues/3815#issuecomment-1426628001

Reply:
Done.
I've removed `bgra8unorm_storage` in WGSL SPEC.